samples/jafu-reactive-data-r2dbc/src/test/java/com/sample/IntegrationTests.java

```diff
@@ -0,0 +1,50 @@
+package com.sample;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+public class IntegrationTests {
+
+	private WebTestClient client = WebTestClient.bindToServer().baseUrl("http://localhost:8181").build();
+
+	private ConfigurableApplicationContext context;
+
+	@BeforeAll
+	void beforeAll() {
+		context = Application.app.run("test");
+	}
+
+	@Test
+	void requestHttpApiEndpoint() {
+		client.get().uri("/").exchange()
+				.expectStatus().is2xxSuccessful()
+				.expectHeader().contentType(MediaType.APPLICATION_JSON_VALUE);
+	}
+
+	@Test
+	public void requestHttpApiEndpointForListingAllUsers() {
+		client.get().uri("/api/user").exchange()
+				.expectStatus().is2xxSuccessful()
+				.expectHeader().contentType(MediaType.APPLICATION_JSON_VALUE)
+				.expectBodyList(User.class)
+				.hasSize(3);
+	}
+
+	@Test
+	public void requestHttpApiEndpointForGettingOneSpecificUser() {
+		client.get().uri("/api/user/blozel").exchange()
```

Comment:
There is a type in the URL :  `"/api/user/b**C**lozel`

Reply:
Thanks it was a mistake a copy from the existing samples. I fudn that the test wasn't complete.
I fixed it